packages/content/src/enrichments.ts

```diff
@@ -376,6 +377,21 @@ const enrichLayers = (
   );
 };
 
+const enrichLicense = (
```

Comment:
I'd defer to @tomwayson on that question.

Reply:
actually since this is sync we can (and should) put it in `itemToContent()` if we can or if not `datasetToContent()` instead of here in `enrichContent()` which is primarily for fetching async enrichments (along w/ a little post-fetch sync processing).

Reply:
ah okay, that makes sense. I moved it to `itemToContent()` 👍